controllers/admin/AdminEmailGeneratorController.php

```diff
@@ -25,21 +28,23 @@ public function __construct()
         $this->context->smarty->assign('emailgenerator', $this->context->link->getAdminLink('AdminEmailGenerator'));
     }
 
-    public function setMedia()
+    public function setMedia($isNewTheme = false)
     {
         $this->addCSS(_PS_MODULE_DIR_.'ps_emailgenerator/css/emailgenerator.css');
         parent::setMedia();
     }
 
     public function processIndex()
     {
+        global $LOCALE;
```

Comment:
If this global wasn't used in the main class, I'd have requested to change it. ;)